core/src/main/java/org/geysermc/geyser/network/GameProtocol.java

```diff
@@ -43,17 +45,13 @@
  */
 public final class GameProtocol {
 
-    // Surprise protocol bump WOW
-    private static final BedrockCodec BEDROCK_V686 = Bedrock_v685.CODEC.toBuilder()
-        .protocolVersion(686)
-        .minecraftVersion("1.21.2")
-        .build();
-
     /**
      * Default Bedrock codec that should act as a fallback. Should represent the latest available
      * release of the game that Geyser supports.
      */
-    public static final BedrockCodec DEFAULT_BEDROCK_CODEC = CodecProcessor.processCodec(BEDROCK_V686);
+    public static final BedrockCodec DEFAULT_BEDROCK_CODEC = CodecProcessor.processCodec(Bedrock_v712.CODEC.toBuilder()
+        .minecraftVersion("1.21.20")
```

Comment:
isn't it already that string?

Reply:
Yeah but we generally just put it since they will likely add a patch version at some point.